### ISSUE TITLE
[Bug] - Unale to Install Ant in Amazon Linux 2023

### ISSUE THREAD

**Issue by tonyswu** [amazonlinux/amazon-linux-2023#530]
**Describe the bug**

The package `ant` lists two version of `amazon-corretto-headless` as dependencies, which results in conflict.

**To Reproduce**
1. Run `dnf install ant`
2. Observe in output:

```
Installing dependencies:
 alsa-lib                                           x86_64                   1.2.7.2-1.amzn2023.0.2                       amazonlinux                   504 k
 ant-lib                                            noarch                   1.10.12-5.amzn2023.0.4                       amazonlinux                   2.0 M
 cairo                                              x86_64                   1.17.6-2.amzn2023.0.1                        amazonlinux                   684 k
 chkconfig                                          x86_64                   1.15-2.amzn2023.0.2                          amazonlinux                   161 k
 dejavu-sans-fonts                                  noarch                   2.37-16.amzn2023.0.2                         amazonlinux                   1.3 M
 dejavu-sans-mono-fonts                             noarch                   2.37-16.amzn2023.0.2                         amazonlinux                   467 k
 dejavu-serif-fonts                                 noarch                   2.37-16.amzn2023.0.2                         amazonlinux                   1.0 M
 fontconfig                                         x86_64                   2.13.94-2.amzn2023.0.2                       amazonlinux                   273 k
 fonts-filesystem                                   noarch                   1:2.0.5-12.amzn2023.0.2                      amazonlinux                   9.5 k
 freetype                                           x86_64                   2.13.0-2.amzn2023.0.1                        amazonlinux                   422 k
 google-noto-fonts-common                           noarch                   20201206-2.amzn2023.0.2                      amazonlinux                    15 k
 google-noto-sans-vf-fonts                          noarch                   20201206-2.amzn2023.0.2                      amazonlinux                   492 k
 graphite2                                          x86_64                   1.3.14-7.amzn2023.0.2                        amazonlinux                    97 k
 harfbuzz                                           x86_64                   7.0.0-2.amzn2023.0.1                         amazonlinux                   868 k
 java-17-amazon-corretto-headless                   x86_64                   1:17.0.9+8-1.amzn2023.1                      amazonlinux                    91 M
 java-21-amazon-corretto-headless                   x86_64                   1:21.0.1+12-1.amzn2023.1                     amazonlinux                    97 M

```

3. Actually installing results in conflict:
```
Error: Transaction test error:
  file /usr/lib/.build-id/0a/66f003181295d8e3399ebbb4b23f5d79fdb311 from install of java-17-amazon-corretto-headless-1:17.0.9+8-1.amzn2023.1.x86_64 conflicts with file from package java-21-amazon-corretto-headless-1:21.0.1+12-1.amzn2023.1.x86_64
  file /usr/lib/.build-id/12/374a2bd17b32bed238a74596c525592331392a from install of java-17-amazon-corretto-headless-1:17.0.9+8-1.amzn2023.1.x86_64 conflicts with file from package java-21-amazon-corretto-headless-1:21.0.1+12-1.amzn2023.1.x86_64
```

**Comment by nmeyerhans** [comment]
Confirmed.  The conflict is between the `java-17-amazon-corretto-headless` and `java-21-amazon-corretto-headless` packages.  For the moment, you can exclude the corretto 21 packages:

`dnf install ant -x java-21-amazon-corretto-headless`

**Comment by nmeyerhans** [comment]
This is resolved as of the 2023.2.20231113 release.
